packages/Webkul/Admin/src/Mail/Order/CanceledNotification.php

```diff
@@ -21,7 +21,7 @@ public function __construct(public $order)
     public function build()
     {
         return $this->from(core()->getSenderEmailDetails()['email'], core()->getSenderEmailDetails()['name'])
-            ->to($this->order->customer_email, $this->order->customer_full_name)
+            ->to(core()->getAdminEmailDetails()['email'], core()->getAdminEmailDetails()['name'])
```

Comment:
No Need to changes here.

Reply:
@amit-webkul,
If we don't change here, the cancellation email will be sent to the customer twice, and the admin will not receive it.

Reply:
Here, mail is delivered to a single customer. But when we look from an admin perspective, we need to add these changes.

**Thank you for this discussion.**